### PR TITLE
Add composer.json for use in PHP projects

### PR DESCRIPTION
**Pull request by legoktm**
So MediaWiki core can depend on and use it.

Signed-off-by: Kunal Mehta <legoktm@member.fsf.org>

(Also where is the bug tracker for this project?)

### CODE REVIEW

**Comment by reedy** [comment]
Could even bring https://packagist.org/packages/mustangostang/spyc in via composer instead of committing the php file :)

**Comment by legoktm** [comment]
Oh, I didn't even notice that. Though I think I remember Nikerabbit saying that he preferred symfony's yaml parser now?

I don't think this library should depend on any yaml parser, it should just be a data repository, and then in core we can have a wrapper or something that checks if the yaml extension is usable, otherwise will fall back to symfony/yaml (or some other pure PHP library)

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "name": "wikimedia/language-data",
+    "description": "Language data and utilities",
+    "license": "GPL-2.0-or-later OR MIT",
+    "authors": [
```
Comment (Nikerabbit) [inline composer.json]: Can we have the regular milkshake authors here? This library used to be part of jquery.uls.

**Comment by Nikerabbit** [comment]
For our yaml file it doesn't really matter: all yaml parsers should parse it in the same way. It's only needed for converting to json anyway.

File: composer.json (hunk 1 above)
Comment (legoktm) [reply composer.json]: Is https://github.com/wikimedia/jquery.uls/blob/master/CREDITS the correct list?

Comment (Nikerabbit) [reply composer.json]: It's the correct list we have for Milkshake, except that I don't think it has ever been reviewed to check if some new contributors should be included in it. I guess one could `git log` the file history in ULS to come up with a more accurate list, or add just the current official maintainers from the Language team.

**Comment by santhoshtr** [comment]
> (Also where is the bug tracker for this project?)

https://github.com/wikimedia/language-data/issues
(sorry, it was somehow disabled in github settings. corrected)

**Comment by cscott** [comment]
Cross-reference: in https://github.com/wikimedia/language-data/pull/36#issuecomment-432356800 @Nikerabbit mentions that PHP support for this library is included in https://phabricator.wikimedia.org/T190129

**Comment by Abijeet** [comment]
I think this PR can be closed.

**Comment by Abijeet** [comment]
> I think this PR can be closed.

To add some context: This has already been added - https://github.com/wikimedia/language-data/pull/79/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780

**Comment by Nikerabbit** [comment]
Sorry Kunal, this got done in another commit.

**Pull request closed without merging**